benchmarks/README.md

```diff
@@ -33,10 +33,14 @@
 | 6.0.0 GA                   | Single node    | 2022/09/25 | c5.xlarge (4 core) | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.0.0/6.0.0_single-node_4-core_jdk-11.md)              |
 | 6.0.0 GA                   | 2-node cluster | 2022/09/27 | c5.large (2 core)  | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.0.0/6.0.0_two-node_2-core_jdk-11.md)                 |
 | 6.0.0 GA                   | 2-node cluster | 2022/09/28 | c5.xlarge (4 core) | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.0.0/6.0.0_two-node_4-core_jdk-11.md)                 |
-| 6.1.0 GA                   | Single node    | 2023/02/01 | c5.large (2 core)  | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.1.0/6.1.0_single-node_2-core_jdk-11.md)              |
-| 6.1.0 GA                   | Single node    | 2023/02/01 | c5.xlarge (4 core) | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.1.0/6.1.0_single-node_4-core_jdk-11.md)              |
-| 6.1.0 GA                   | 2-node cluster | 2023/02/01 | c5.large (2 core)  | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.1.0/6.1.0_two-node_2-core_jdk-11.md)                 |
-| 6.1.0 GA                   | 2-node cluster | 2023/02/01 | c5.xlarge (4 core) | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.1.0/6.1.0_two-node_4-core_jdk-11.md)                 |
+| 6.1.0 GA - V1                  | Single node    | 2023/02/01 | c5.large (2 core)  | db.m4.2xlarge     | JDK 11      | [:arrow_upper_right:](6.1.0/6.1.0_single-node_2-core_jdk-11.md)              |
```

Comment:
links needs to be fixed

Reply:
Resolved with https://github.com/wso2/performance-is/pull/156/commits/06d35b046a65eab461d21c01e0250733f3b226b1